gax-httpjson/src/main/java/com/google/api/gax/httpjson/ManagedHttpJsonChannel.java

```diff
@@ -53,81 +48,80 @@
   private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
 
   private final Executor executor;
-  private final HttpTransport httpTransport;
   private final String endpoint;
   private final JsonFactory jsonFactory;
   private final ImmutableList<HttpJsonHeaderEnhancer> headerEnhancers;
+  private final HttpTransport httpTransport;
+
+  private boolean isTransportShutdown;
 
   private ManagedHttpJsonChannel(
       Executor executor,
-      HttpTransport httpTransport,
       String endpoint,
       JsonFactory jsonFactory,
       List<HttpJsonHeaderEnhancer> headerEnhancers) {
     this.executor = executor;
-    this.httpTransport = httpTransport;
     this.endpoint = endpoint;
     this.jsonFactory = jsonFactory;
     this.headerEnhancers = ImmutableList.copyOf(headerEnhancers);
+
+    HttpTransport httpTransport = new NetHttpTransport();
+    this.httpTransport = httpTransport;
```

Comment:
Is there a reason we don't want to allow the http transport to be passed in? We can default it if it isn't provided.

Reply:
It's back now, with a default value.